cmd/deploy/deploy.go

```diff
@@ -286,18 +287,28 @@ func (dc *DeployCommand) RunDeploy(ctx context.Context, deployOptions *Options)
 		return oktetoErrors.ErrDeployCantDeploySvcsIfNotCompose
 	}
 
+	// We need to create a client that doesn't go through the proxy to create
```

Comment:
I moved this logic up so that `cwd, err := os.Getwd()` is closer to the code using `cwd`

Reply:
This is just the k8s client, we are not calling to get any configmap on this call

Reply:
saying that because the comment of the fn

Reply:
@AdrianPedriza it's not a change that I've introduced so I am not sure why the comment is there